torch/_dynamo/variables/base.py

```diff
@@ -458,7 +484,13 @@ def call_method(
                 not other.is_python_constant()
                 or tx.output.side_effects.has_pending_mutation(other)
             ):
-                unimplemented(f"call_method {self} {name} {args} {kwargs}")
+                unimplemented_v2(
+                    gb_type="Comparison failed",
```

Comment:
Make this a little bit more descriptive? "Builtin `operator.*` comparison with constant `self` failed"

Reply:
@williamwen42 Thanks for your careful review! I didn't mark gb_type as an f-string, that's right?